src/main/java/com/google/api/codegen/util/SymbolTable.java

```diff
@@ -30,6 +30,23 @@
   private final Set<String> symbolTable = new HashSet<>();
 
   /**
+   * Returns a new SymbolTable seeded with all the words in seed.
+   *
+   * For example, if seed is {"int"}, a subsequent call to {@link
+   * #getNewSymbol(String)} for "int" will return "int2".
```

Comment:
Something to think about: it might be nice to be able to pass in a hint parameter for when renaming is needed. For example, this could be the resource name. Thus, `int` -> `resourceInt`. Or, more realistically for Go (prompted by #550), `range` -> `listRange`. These would read better and be less confusing than `int2` and `range2`.

I'm OK deferring this, but if so, please file a tracking issue.